packages/firestore/src/platform_node/node_platform.ts

```diff
@@ -83,4 +87,14 @@ export class NodePlatform implements Platform {
 
     return randomBytes(nBytes);
   }
+
+  /**
+   * On Node, only supported data source is a `Uint8Array` for now.
+   */
+  toByteStreamReader(source: unknown): ByteStreamReader {
+    if (source instanceof Uint8Array) {
```

Comment:
I suggest you use this pattern here: https://github.com/firebase/firebase-js-sdk/blob/e67affba53a53d28492587b2f60521a00166db60/packages/firestore/src/api/blob.ts#L81

If it doesn't apply here, then you should invoke it at the callsite (the API that the end user calls) and add a `debugAssert` here.

Reply:
Done.